tests/test_app.py

```diff
@@ -0,0 +1,26 @@
+from importlib import reload
+
+import lambdas
+
+
+def test_app_configures_sentry_if_dsn_present(
+    caplog,
+    monkeypatch,
+):
+    monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
+    reload(lambdas)
```

Comment:
The usage of `reload(...)` in these tests makes me a little nervous.  I wonder if `lambdas.py` can be reworked such that it would reflect changes to monkey-patched env vars for testing, removing the need for this?

Reply:
One thought that comes to mind is initializing Sentry from either a standalone function in `config.py` ([example from HRQBClient](https://github.com/MITLibraries/hrqb-client/blob/main/hrqb/config.py#L74-L80)), or even as part of the class.

Then, we could just call this function from the actual lambda handler function which would init Sentry.

Reply:
Please take a look at the updates in [config.py](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1/commits/f15fa67df96f8bf400651f41c68ceedf7d9f0417), [app.py](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1/commits/d15cd2a18a626d11f5ee206c375f70d9ab55dbe9#diff-55f12a9f17b4ba61d3e37852150b7d78b8aedbe3629bc4ce8173d4b572297a54), and [lambdas.py](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/1/commits/2236bad54c1f5c142b8aa7db897efc6d017afe11#diff-8b63c8f1f6bfc218ee5ff375b538bdcf99123412580182042f7a154151bddef8R22-R26).